addons/sourcemod/scripting/include/saxtonhale.inc

```diff
@@ -239,6 +239,13 @@ methodmap SaxtonHaleBase
 		public native set(bool val);
 	}
 	
+	// Retrieve/Set whether the boss' max health calculation should consider currently-living players or the playercount at the beginning of the round
+	property bool bHealthPerPlayerAliveOnly
```

Comment:
I think `bHealthPerPlayerAlive` without `Only` is best to name it, there already property similarly named `iHealthPerPlayer`